kubernetes/zenko/charts/cloudserver/dashboards/cloudserver.json

```diff
@@ -296,54 +196,56 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "nodejs_active_handles_total{instance=\"$pod\"}",
+          "expr": "sum(increase(cloud_server_http_request_size_bytes_sum{app=\"cloudserver\"}[30d]))",
```

Comment:
Is there a particular reason why this metric is accumulated over 30 days while the "data ingested into Zenko" metric is cumulated over 14 days only? If not, I would think it's better to have comparable metrics using the same accumulation period.

Reply:
Oops, yeah this was an accident. I first increased them to 14d each and then 30d in my testing but missed this. I'll fix this